Finjector.Web/ClientApp/src/components/Teams/NameAndDescriptionForm.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+import { NameAndDescriptionModel } from "../../types";
+
+interface Props {
+  initialValues?: NameAndDescriptionModel;
+  onSubmit: (formData: NameAndDescriptionModel) => void;
+}
+
+export default function FormComponent(props: Props) {
+  const { initialValues = { name: "", description: "" }, onSubmit } = props;
+  const [formData, setFormData] = useState(initialValues);
+  const [loading, setLoading] = useState(false);
+
+  const handleChange = (e: any) => {
+    setFormData({ ...formData, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = async (e: any) => {
+    e.preventDefault();
+    setLoading(true);
+    await onSubmit(formData);
+    setLoading(false);
+  };
+
+  return (
+    <form className="form" onSubmit={handleSubmit}>
+      <div className="mb-3">
+        <label htmlFor="name" className="form-label">
+          Name:
+        </label>
+        <input
+          type="text"
+          className="form-control"
+          id="name"
+          name="name"
+          value={formData.name}
+          onChange={handleChange}
+          required
+          maxLength={50}
+        />
+        <div className="invalid-feedback">Please provide a name.</div>
```

Comment:
i don't ever see this. just get this: 
![image](https://github.com/ucdavis/finjector/assets/27025973/c9dfade6-a0ce-4068-8860-80ab9ebe6ef6)
also, should these all be Reactstrap?